setup/includes/upgrades/common/3.0.0-update-menu-entries.php

```diff
@@ -10,8 +10,16 @@
 
 /** @var modMenu $formCustomization */
 $formCustomization = $modx->getObject(modMenu::class, ['text' => 'bespoke_manager']);
+
 if ($formCustomization) {
-    $formCustomization->set('text', 'form_customization');
-    $formCustomization->set('description', 'form_customization_desc');
-    $formCustomization->save();
+    $newData = $formCustomization->toArray();
+    $newData['text'] = 'form_customization';
+    $newData['description'] = 'form_customization_desc';
+
+    $formCustomization->remove();
```

Comment:
I think it's better to remove the old entry after the new one is created/updated.